.github/workflows/ci-goreleaser.yaml

```diff
@@ -35,7 +35,8 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v3
```

Comment:
this and the other actions will be good to replace that to use the git hashes to avoid any issues if the tag mutate and break or other possible security implications

Reply:
Not sure -- this could very well be a branch, so using "v3" would just use the latest state of that branch.